js/src/components/google-combo-account-card/connect-google-combo-account-card.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement, useState } from '@wordpress/element';
+import { CheckboxControl } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { glaData } from '.~/constants';
+import AccountCard, { APPEARANCE } from '.~/components/account-card';
+import AppButton from '.~/components/app-button';
+import readMoreLink from '../google-account-card/read-more-link';
+import useGoogleConnectFlow from '../google-account-card/use-google-connect-flow';
+import AppDocumentationLink from '../app-documentation-link';
+
+/**
+ * @param {Object} props React props
+ * @param {boolean} props.disabled
+ * @fires gla_google_account_connect_button_click with `{ action: 'authorization', context: 'reconnect' }`
+ * @fires gla_google_account_connect_button_click with `{ action: 'authorization', context: 'setup-mc' }`
+ * @fires gla_documentation_link_click with `{ context: 'setup-mc-accounts', link_id: 'required-google-permissions', href: 'https://woocommerce.com/document/google-for-woocommerce/get-started/setup-and-configuration/#required-google-permissions' }`
+ * @fires gla_documentation_link_click with `{ context: 'setup-mc-accounts', link_id: 'google-mc-terms-of-service', href: 'https://support.google.com/merchants/answer/160173' }`
+ * @fires gla_documentation_link_click with `{ context: 'setup-ads', link_id: 'google-ads-terms-of-service', href: 'https://support.google.com/adspolicy/answer/54818' }`
+ */
+const ConnectGoogleComboAccountCard = ( { disabled } ) => {
+	const pageName = glaData.mcSetupComplete ? 'reconnect' : 'setup-mc';
+	const [ handleConnect, { loading, data } ] =
+		useGoogleConnectFlow( pageName );
+	const [ termsAccepted, setTermsAccepted ] = useState( false );
+
+	const checkboxId = 'gla-account-card-terms-conditions';
+
+	return (
+		<AccountCard
+			appearance={ APPEARANCE.GOOGLE }
+			className={ 'google-combo-account-card' }
+			disabled={ disabled }
+			alignIcon="top"
+			description={
+				<>
+					{ __(
```

Comment:
@ankitrox Let's wrap this string within a paragraph tag.

Reply:
@asvinb If we wrap it in paragraph tag, it will produce invalid HTML because checkbox control will yield `div` and we should not have div inside `<p>` tag.

Reply:
Apologies. I was unclear @ankitrox . Only the text `Required to sync with Google Merchant Center and Google Ads.` should be wrapped within a `p` tag.